molgenis-navigator/src/main/frontend/test/unit/specs/store/store.spec.js

```diff
@@ -1,100 +1,95 @@
-import * as api from 'molgenis-api-client'
+import api from '@molgenis/molgenis-api-client'
 import store from 'src/store'
 
-describe('store', function () {
-  describe('initial state', function () {
+describe('store', () => {
```

Comment:
mocha warns against using () = > style because of problems with scoping, use function ()